fiaas_deploy_daemon/specs/v3/defaults.yml

```diff
@@ -56,6 +56,7 @@ metrics:
     path: /_/metrics # Path to prometheus-metrics
   datadog:
     enabled: false # if true, attach a datadog sidecar
+    tags: # A list of custom tags to add to all metrics exposed by the pods
```

Comment:
tags: {} # A map of custom tags to add to all metrics exposed by the pods

Reply:
Thanks!, fixed